suites/reef/cephadm/tier4-bootstrap-with-ca-signes-ssh-keys.yaml

```diff
@@ -0,0 +1,91 @@
+#===============================================================================================
+# Conf: conf/quincy/cephadm/tier-0.yaml
+# Test cases for cluster service operation
+#    - Bootstrap
+#    - Perform stop operation for Mon and Mgr service
+#
+#===============================================================================================
+tests:
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: setup pre-requisites
+
+  - test:
+      abort-on-fail: true
+      desc: Setup CA signed ssh keys
+      module: test_setup_ca_signed_ssh_keys.py
+      polarion-id: CEPH-83589818
+      name: Sets CA signed ssh keys on the ceph nodes
+
+  - test:
+      abort-on-fail: true
+      config:
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+          - config:
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.1
+              args:
+                placement:
+                  label: rgw
+          - config:
+              args:
+                - "ceph fs volume create cephfs"
+              command: shell
+          - config:
+              args:
+                placement:
+                  label: mds
+              base_cmd_args:
+                verbose: true
+              command: apply
+              pos_args:
+                - cephfs
+              service: mds
+          - config:
+              args:
+                - "ceph osd pool create rbd"
+              command: shell
+          - config:
+              args:
+                - "rbd pool init rbd"
+              command: shell
+      desc: bootstrap and deploy services.
+      destroy-cluster: false
+      polarion-id: CEPH-83573713
+      module: test_cephadm.py
```

Comment:
Why separate test case for one additional step? Can't it be part of bootstrap test case?

Reply:
So the logic here is Vaibhav, we need to Setup CA signed ssh keys  (line num 15) prior to bootstrap.
And the bootstrap should succeed. Since this is a separate scenario, created a new suite

Reply:
@pranavprakash20 issue is not with the test suite. We are adding a new test file for a single step.

I suggest to add this step in test_cephadm.py and pass parameter to call required step.

Reply:
@vamahaja Noted. It make sense to add to test_cephadm.py
Add a new param for the ca ssh key. Please review